test/unit/errors.spec.js

```diff
@@ -1,24 +1,20 @@
-/* global $,postscribe,test,ok,start */
-/* eslint-disable no-var */
+import postscribe from '../../dist/postscribe';
 
-$(document).ready(function() {
-
-  module('errors');
-
-  function testError(name, html) {
-    test(name, function() {
-      var oldOnError = window.onerror;
+describe('errors', function() {
+  const testError = (name, html) => {
+    it(name, () => {
```

Comment:
Again, I'd tend to factor out the `it` part and call this something like `handlesError`, so you'd do `it('remote script 404', handlesError('<script...'));`.  Makes it a bit easier to grep for `it`.